src/docs/spell.ok

```diff
@@ -11,6 +11,7 @@ Adler's
 Atomicity
 BLOBs
 BLRrVv
+BLmRrSVv
```

Comment:
Interestingly, I was going to suggest removing `RVv` but that is not in `spell.ok`. But `RrVv` is and certainly applied to this. I'm not sure how `RVv` wasn't getting flagged. In any case, you can probably also remove `RrVv` from this file.